readme.md

```diff
@@ -34,9 +34,11 @@ $ aws-cost --help
 
     -k, --access-key [key]         AWS access key
     -s, --secret-key [key]         AWS secret key
+    -t, --session-token [key]      AWS session token
     -r, --region [region]          AWS region (default: us-east-1)
     -a, --role-arn [arn]           AWS role ARN to assume
 
+    --target-account [id]          Account ID to see cost
```

Comment:
Troublesome, but no good idea comes to mind 🤔

Reply:
I added https://github.com/route06/aws-cost-cli/pull/20/commits/0eb8cb76888fef0ee10b106a2f818a076ad397a1